packages/widget-message-meet/src/components/widget-data-input/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/no-set-state, global-require */
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+import SparkLogo from '../spark-logo';
+
+class WidgetDataInput extends Component {
+  constructor(props) {
+    super(props);
+    const {accessToken, toPersonEmail} = props;
+    const error = false;
+    this.state = {
+      accessToken,
+      error,
+      toPersonEmail
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleAccessTokenChange = (e) => this.setState({accessToken: e.target.value});
+    this.handleEmailChange = (e) => this.setState({toPersonEmail: e.target.value});
```

Comment:
Oh yea, I didn't want to pollute the store with something that will probably only be used with demos.